go.mod

```diff
@@ -1,33 +1,33 @@
 module github.com/kube-vip/kube-vip
 
-go 1.18
+go 1.19
```

Comment:
Should it be 1.20 since you updated dockerfile to use 1.20 as base image

Reply:
not necessarily, as kube-vip would still build with 1.19